Update Roadmap, add Orchid popup (#82)
Vanilla-OS/website: roadmap.md

                    description: "The development of Vanilla OS 22.10 will be frozen until the next release.",
                    className: "timeline-green"
                },
                {
                    date: "26 February 2023",
                    title: "Manual Partitioning Support",
                    description: "Added Manual Partitoning Support to Installer",
                    className: "timeline-dimmed-green"
                },
                {
                    date: "26 February 2023",

>                     title: "Panels removed in Vanilla Control cCnter",

Reviewer: ```suggestion
                    title: "Panels removed in Vanilla Control Center",
```

>                     description: "Drivers and PRIME Profiles panels have been moved to Settings from First Setup",

Reviewer: ```suggestion
                    description: "Drivers and PRIME Profiles panels have been moved to Settings from Vanilla Control Center",
```

                    className: "timeline-dimmed-green"
                },
                {
                    date: "February 2023",
                    title: "Nix Support",
                    description: "Added support for Nix packages in Apx",
                    className: "timeline-dimmed-green"
                },
                {
                    date: "February 2023",
                    title: "Rollback Support",

>                     description: "Added option rollback to previous root partition in ABRoot",

Author: ```suggestion
                    description: "Added option to rollback to previous root partition in ABRoot",
```
